src/App.jsx

```diff
@@ -1,31 +1,35 @@
-import React, { useState, useEffect } from 'react';
-import { getAllData } from './util/index';
+import React, { useState, useEffect } from "react";
+import { getAllData } from "./util/index";
+import CssBaseline from "@mui/material/CssBaseline";
+import { ThemeProvider } from "@mui/material/styles";
+import theme from "./Theme";
```

Comment:
this import does not exist

Reply:
I've cleaned up my App component as it will be relocated to the SearchBar component, which is part of the Navbar component. The original code was added temporarily for review and testing in the browser. Sorry for the confusion